quantinuum-hugr/src/extension/infer/test.rs

```diff
@@ -795,6 +794,28 @@ fn test_cfg_loops() -> Result<(), Box<dyn Error>> {
     Ok(())
 }
 
+#[test]
+#[cfg(feature = "extension_inference")]
+fn test_validate_with_closure() -> Result<(), Box<dyn Error>> {
+    let mut hugr = make_looping_cfg(
+        ExtensionSet::new(),
+        ExtensionSet::singleton(&A),
+        ExtensionSet::singleton(&B),
+    )?;
+    assert_matches!(
+        hugr.validate(&PRELUDE_REGISTRY),
+        Err(ValidationError::ExtensionError(_))
+    );
+
+    let immut = hugr.clone();
+    let soln = infer_extensions(&immut)?;
+    immut.validate_with_extension_closure(soln, &PRELUDE_REGISTRY)?;
```

Comment:
Looks like this should just be `immut.validate(soln, &PRELUDE_REGISTRY)`?

Reply:
?? `validate` takes only one argument, the registry.